### ISSUE TITLE
Can't find certain (new-ish?) components

### ISSUE THREAD

**Issue by nphilipp** [fedora-infra/fedora-packages#410]
When submitting an update in Bodhi for the latest release of `fedmod`, I noticed that typing it into the packages field wouldn't offer it for selection. I was told on IRC that the packages app is the source of components for Bodhi, this is why I'm reporting this here:

- The packages app doesn't find `fedmod` or (random other new package) `perl-Games-Solitaire-Verify`.
- As a consequence, Bodhi doesn't suggest them in the 'Packages' field and builds have to be typed in manually.

**Comment by cverna** [comment]
The consumer is disabled in production since it was corrupting the database (creating multiple entry of the same package) , I have been running the indexing manually once a week, but this week it has problems completing successfully, I ll give it another try later today.

Thanks for the report

**Comment by richardfearn** [comment]
Is it possible to do a manual index soon so that Fedora Packages catches up with recently-created packages?

**Comment by cverna** [comment]
Yes, I am waiting on https://github.com/fedora-infra/fedora-packages/issues/411 to run the index. 

That should happend this week :crossed_fingers:

**Comment by richardfearn** [comment]
@cverna Thank you!

**Comment by Lyude** [comment]
Hi! I was told on https://pagure.io/fedora-infrastructure/issue/7974#comment-581356 that I should note here that one of the packages I maintain, igt-gpu-tools, seems to have the same issue on bodhi

**Comment by tim77** [comment]
My 2 cents. :) For example:

1. **Peek**
    https://apps.fedoraproject.org/packages/peek
    Source: https://src.fedoraproject.org/rpms/peek

2. **CoreCtrl**
    https://apps.fedoraproject.org/packages/corectrl
    Source: https://src.fedoraproject.org/rpms/corectrl

Expected to see like this:

https://apps.fedoraproject.org/packages/firefox

**Comment by cverna** [comment]
Ok all these packages are now available. some are missing the metadata I am not sure why yet but I ll try to check it next week.

**Comment by dreua** [comment]
What's the status here? Currently I can't find [pdfarranger](https://apps.fedoraproject.org/packages/pdfarranger)

**Comment by richardfearn** [comment]
> What's the status here? Currently I can't find [pdfarranger](https://apps.fedoraproject.org/packages/pdfarranger)

Ditto for [perl-App-ccdiff](https://apps.fedoraproject.org/packages/s/perl-App-ccdiff)

**Comment by dreua** [comment]
People actually use this and are confused if a package is not found: http://disq.us/p/2552f0d

**Comment by lcts** [comment]
> What's the status here? Currently I can't find [pdfarranger](https://apps.fedoraproject.org/packages/pdfarranger)

Same issue with [sshguard](https://apps.fedoraproject.org/packages/sshguard)

**Comment by tim77** [comment]
> People actually use this and are confused if a package is not found: http://disq.us/p/2552f0d

Very confused. I am tired notify upstream that their app packaged but when they click on package home page in apps.fedoraproject.org they can't find it and starting asking questions.

**Comment by jamesjer** [comment]
I've got packages that were created in February that can't be found.  @cverna, what is the status of this issue?  Do you need help diagnosing it?  Can you run another manual indexing operation while the problem is being figured out?

**Comment by QuLogic** [comment]
There are packages like `RBTools` that according to koji have existed since _FC12_ yet cannot be found, so I'm not sure this is limited to _new_ components only.

**Comment by kees-closed** [comment]
Some packages are still missing; charliecloud, fwsnort, and many more. `dnf search` works fine, but sometimes a web based version provides some convenience. Thanks in any case.
